distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -1118,15 +1033,34 @@ public static String getSimpleXml(String uri) {
                 ImmutableMap.of("uri", uri));
     }
 
-    public void configureShowInvalidMetacardsReset() throws IOException {
-        configureShowInvalidMetacards("false", "true", getAdminConfig());
-    }
-
-    public void configureFilterInvalidMetacardsReset() throws IOException {
-        configureFilterInvalidMetacards("true", "false", getAdminConfig());
+    /**
+     * Setting configurations is performed asynchronously and there is no way to check if the configured
+     * bean has received a configuration update. This method provides a best effort workaround by retrying
+     * the test/assertions with a slight delay in between tries in an attempt to let the configuration
+     * thread catch up
+     *
+     * @param runnable
+     */
+    public void testWithRetry(Runnable runnable) {
```

Comment:
if this logic gets added to any more classes, should it be moved to one of the common itest classes?